multisig/simulations/vip-010/vip-010-ethereum/simulations.ts

```diff
@@ -134,6 +138,12 @@ forking(19175701, () => {
     }
   });
 
+  describe("Secondary wstETH oracle", () => {
+    it("should revert for uncofigured stETH price", async () => {
+      await expect(wstETHNonEquivalentOracle.getPrice(WSTETH)).to.be.revertedWith("invalid resilient oracle price");
+    });
+  });
+
```

Comment:
Why we are asserting wstETHNonEquivalentOracle here? On the other hand we configured Equivalence one.

moreover, without setting Equivalence oracle config in Resilient tests are passing. Shall we assert this checks from ResilientOracle?